### ISSUE TITLE
"prod" flavor not working on iOS

### ISSUE THREAD

**Issue by ThomasEcalle** [AngeloAvv/flutter_flavorizr#48]
Hi,

First of all, thank you a lot for this package that is amazing in flavor handling.

I have tried this configuration :

```
flavorizr:
  app:
    android:
      flavorDimensions: "flavor-type"
    ios:
  flavors:
    dev:
      app:
        name: "App-dev"
      android:
        applicationId: "com.example.dev"
      ios:
        bundleId: "com.example.dev"

    preprod:
      app:
        name: "App-preprod"
      android:
        applicationId: "com.example.preprod"
      ios:
        bundleId: "com.example.preprod"

    prod:
      app:
        name: "App-prod"
      android:
        applicationId: "com.example.prod"
      ios:
        bundleId: "com.example.prod"
```

Everything works on both Android & iOS except `prod` flavor on iOS.

Building prod flavor on iOS throw this error :

```
The Xcode project defines build configurations: Debug, Release, Profile, Debug dev, Profile dev, Release dev, Debug preprod, Profile preprod, Release preprod, Debug prod, Profile prod, Release prod
Flutter expects a build configuration named Debug-prod or similar.
Open Xcode to fix the problem:
  open ios/Runner.xcworkspace
1. Click on "Runner" in the project navigator.
2. Ensure the Runner PROJECT is selected, not the Runner TARGET.
3. Click the Editor->Add Configuration->Duplicate "Debug" Configuration.

   If this option is disabled, it is likely you have the target selected instead
   of the project; see:
   https://stackoverflow.com/questions/19842746/adding-a-build-configuration-in-xcode

   If you have created a completely custom set of build configurations,
   you can set the FLUTTER_BUILD_MODE=debug
   in the .xcconfig file for that configuration and run from Xcode.

4. If you are not using completely custom build configurations, name the newly created configuration debug.
Could not build the application for the simulator.
Error launching application on iPhone 12 Pro Max.
```

"dev" and "preprod" flavors are working well but not "prod".

Does someone have an idea of the issue ?

Thank you in advance !

**Comment by tripuraritiwari** [comment]
same with me.

**Comment by PhanithNoch** [comment]
anyone help please i can't understand how we can add file google-services.json and GoogleService-Info.plist to multiple environments

**Comment by AngeloAvv** [comment]
Hi @ThomasEcalle , I cannot reproduce your issue. I pasted your configuration in my pubspec.yaml, I'm using flutter_flavorizr 2.0.0. This is the output I got once I try to build the prod flavor:

```
angeloavv@MBP-di-Angelo untitled % flutter build ios --flavor prod -t lib/main-prod.dart 

Downloading ios tools...                                            4,6s
Downloading ios-profile tools...                                    3,3s
Downloading ios-release tools...                                   21,7s
Building com.example.untitled for device (ios-release)...
Signing iOS app for device deployment using developer identity: "blablabla"
Running Xcode build...                                          
Xcode build done.                                           51,2s
Built /Users/angeloavv/FlutterProjects/untitled/build/ios/iphoneos.
angeloavv@MBP-di-Angelo untitled % 
```

I also tried to run the flavor without problems. Just make sure you have opened an iOS Simulator before:
```
angeloavv@MBP-di-Angelo untitled % flutter run --flavor prod -t lib/main-prod.dart

Launching lib/main-prod.dart on iPad Pro (12.9-inch) (4th generation) in debug mode...
Running Xcode build...                                          
Xcode build done.                                           35,8s
Syncing files to device iPad Pro (12.9-inch) (4th generation)...        219ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

💪 Running with sound null safety 💪

An Observatory debugger and profiler on iPad Pro (12.9-inch) (4th generation) is available at: http://127.0.0.1:52277/uKQLtcbmuOo=/
Activating Dart DevTools...                                         5,5s
The Flutter DevTools debugger and profiler on iPad Pro (12.9-inch) (4th generation) is available at: http://127.0.0.1:9102?uri=http%3A%2F%2F127.0.0.1%3A52277%2FuKQLtcbmuOo%3D%2F


```
